src/main/java/com/synopsys/integration/alert/channel/azure/boards/web/AzureBoardsCustomFunctionAction.java

```diff
@@ -155,11 +155,13 @@ private OAuthEndpointResponse createErrorResponse(String errorMessage) {
         if (StringUtils.isNotBlank(fieldModel.getId())) {
             if (authorizationManager.hasWritePermission(ConfigContextEnum.GLOBAL.name(), azureBoardsChannelKey.getUniversalKey())) {
                 Long id = Long.parseLong(fieldModel.getId());
-                return Optional.ofNullable(configActions.updateConfig(id, fieldModel));
+                ActionResponse<FieldModel> response = configActions.update(id, fieldModel);
```

Comment:
Won't `configActions.update` check write permissions? Should the if-statement check if the id is present to determine whether to create or update?

Reply:
yes